AWSAuthSDK/AWSAuthSDK.xcodeproj/xcshareddata/xcschemes/AWSMobileClient.xcscheme

```diff
@@ -28,15 +28,6 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
       codeCoverageEnabled = "YES">
-      <MacroExpansion>
```

Comment:
What does this change reflect?

Reply:
not sure why this was removed, adding back